src/plugins/application_config/server/opensearch_config_client.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { IScopedClusterClient, Logger } from '../../../../src/core/server';
+
+import { ConfigurationClient } from './types';
+
+export class OpenSearchConfigurationClient implements ConfigurationClient {
+  private client: IScopedClusterClient;
+  private configurationIndexName: string;
+  private readonly logger: Logger;
+
+  constructor(
+    inputOpenSearchClient: IScopedClusterClient,
```

Comment:
the input prefix of method parameters seems redundant

Reply:
+1

Reply:
Updated the variable name to look clean